modules/installation-configuration-parameters.adoc

```diff
@@ -1,54 +1,110 @@
+to your {product-title} cluster components.
+|A fully-qualified domain or subdomain name, such as `dev.example.com`.
+
+|`name`
+|The name of your cluster.
+|A string that contains uppercase or lowercase letters, such as `dev`.
+
+|`region`
+|The name of your cluster.
+|A valid AWS region, such as `us-east-1`.
 
-* aws
-* azure
-* openstack
+|`pullSecret`
+|The pull secret that you obtained from link:try.openshift.com[the OpenShift start page].
+|
+[source,json]
+----
+{  
+   "auths":{  
+      "cloud.openshift.com":{  
+         "auth":"b3Blb=",
+         "email":"you@example.com"
+      },
+      "quay.io":{  
+         "auth":"b3Blb=",
+         "email":"you@example.com"
+      }
+   }
+}
+----
 
-|`base-domain`
-|The base domain of your cloud provider. This value is used to <do a thing>.
-|A FQDN, such as `dev.example.com`.
+|`sshKey`
+|The SSH key to use to access your cluster machines.
+|A valid public SSH key.
 |====
 
-.All installation parameters
-[cols=".^3,.^5,.^5a,.^5",options="header"]
+
+.Optional AWS platform parameters
+[cols=".^2,.^3,.^3a",options="header"]
 |====
-|Parameter|Description|Value format|Default value
+|Parameter|Description|Values
 
-|`provider`
-|The cloud provider to host the cluster.
-a|Enter one of the following values:
+|`machines.platform.aws.iamRoleName`
+|The IAM role assigned to the machines in the pool.
+|
+
+|`machines.platform.aws.rootVolume.iops`
+|The Input/Output Operations Per Second (IOPS) that is reserved for the root volume.
+|Integer, for example `4000`
+
+|`machines.platform.aws.rootVolume.size`
+|The size in GiB of the root volume.
+|Integer, for example `500`
+
+|`machines.platform.aws.rootVolume.type`
+|The instance type of the root volume.
+|Valid link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html[AWS EBS instance type],
+such as `io1`.
+
+|`machines.platform.aws.type`
+|The EC2 instance type for the control plane machines.
+|Valid link:https://aws.amazon.com/ec2/instance-types/c5/[AWS instance type],
+such as `c5.9xlarge`.
+
+|`machines.platform.aws.zones`
+|The availability zones where the installation program creates machines for this
+pool.
+|Valid AWS availability zone, such as `us-east-1c`.
 
-* aws
-* azure
-* openstack
-| n/a
+|`platform.aws.region`
+|The AWS region that the installation program creates resources in.
+|Valid link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS region],
+such as `us-east-1`
 
-|`base-domain`
-|The base domain of your cloud provider. This value is used to <do a thing>.
-|A FQDN, such as `dev.example.com`.
-|n/a
+|`platform.aws.userTags`
+|A map of keys and values that the installation program adds as tags to all
+resources that it creates.
+|Key value pairs in the `<key>: <value>` format. For more information about
+AWS IAM tags, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_tags.html[Tagging IAM Entities]
+in the AWS documentation.
```

Comment:
Why call out IAM tags here?  If we wanted to link to docs, I'd have expected something generic like [this][1].

[1]: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html

Reply:
Thank you for the better link! Is the description still acceptable, or is there better information to provide?

Reply:
> Is the description still acceptable, or is there better information to provide?

Your [current wording][1] looks fine to me.  Was there more information you were fishing for?

[1]: https://github.com/openshift/openshift-docs/commit/3a0157c04c643de1e77c14b6ada661660ade6559#diff-7492465e78e37df8219ae1117519d026R106

Reply:
The link was specifically to IAM tags before, but we made it more general. I wanted to be sure that the description was still fair.

Reply:
> The link was specifically to IAM tags before, but we made it more general. I wanted to be sure that the description was still fair.

Looks like you made the description more general too, so it matches the link.  And `userTags` itself is generic (not IAM-specific), so I think we're all good.